### PR TITLE
Run `npm install` before running emscripten tests

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by dschuff** [comment]
Looks like this isn't working: https://ci.chromium.org/p/emscripten-releases/builders/try/linux/b8890072294421298416
although that's odd because we put the node Bin dir in the path: https://github.com/WebAssembly/waterfall/blob/master/src/build.py#L170

File: src/build.py [hunk 1]
```diff
@@ -1615,6 +1615,8 @@ def ExecuteEmscriptenTestSuite(name, tests, config, outdir, warn_only=False):
   buildbot.Step('Execute emscripten testsuite (%s)' % name)
   Mkdir(outdir)
   try:
+    proc.check_call([os.path.join(NodeBinDir(), 'npm'), 'install'],
```
Comment (dschuff) [inline src/build.py]: also. does this need to be wrapped in `Executable()`?

Comment (sbc100) [reply src/build.py]: Yup .. I think so.

**Comment by sbc100** [comment]
I don't get it either..

**Pull request merged**